easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033_update_gwt.patch

```diff
@@ -0,0 +1,22 @@
+--- a/dependencies/common/install-gwt	2020-02-04 11:16:49.907923640 +0000
```

Comment:
Please add a description and authorship of this patch.

Reply:
Done: https://github.com/easybuilders/easybuild-easyconfigs/pull/9826/commits/06364476f53f237afd71f77501fc11b6d6ed4c63

Note: Patch originally extracted from https://github.com/bear-rsg/easybuild-easyconfigs/pull/52 and suggested by @edmondac